pom.xml

```diff
@@ -85,7 +85,7 @@
         <dependency>
             <groupId>org.jboss.aerogear.unifiedpush</groupId>
             <artifactId>unifiedpush-push-model</artifactId>
-            <version>1.1.0-alpha.1</version>
+            <version>1.1.0-alpha.2-SNAPSHOT</version>
```

Comment:
You mean removing the `-snapshot` ? Probably ... But for now since alpha.2 only exist as `snapshot` I added it.